.circleci/config.yml

```diff
@@ -161,27 +161,3 @@ workflows:
           filters:
             branches:
               only: master
-
-      - heroku/deploy-via-git:
-          app-name: simple-hub-server-staging
```

Comment:
Is this also being extracted?

Reply:
Oops! These weren't meant to be removed, reverting.